### ISSUE TITLE
Email notifications show unchecked checkboxes

### ISSUE THREAD

**Issue by d-karstens** [verbb/formie#429]
**Description**
Users are getting email notifications listing every checkbox field option, rather than just the ones ticked

**Steps to reproduce**
1. Create form with a checkbox field with a few options
2. Make a submission with just one of them checked
3. Notification email shows all options

<img width="913" alt="Screen Shot 2021-04-12 at 10 57 10 am" src="https://user-images.githubusercontent.com/21274925/114329236-f8a96380-9b7d-11eb-81ce-c9230836b867.png">

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Both
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.27
- Craft version: 3.6.11.2
- Multi-site: No

**Additional context**

**Comment by jasondibenedetto** [comment]
Any update on this one?

**Comment by d-karstens** [comment]
As a temporary fix, we've used "All non-empty fields", which seems to display them correctly, but this still happens when just using the checkbox variable

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.27",
  "...": "..."
}
```

Then run `composer update`.

**Comment by d-karstens** [comment]
Thank you!

**Comment by engram-design** [comment]
Fixed in [1.4.0](https://github.com/verbb/formie/releases/tag/1.4.0)
